sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -97,9 +98,15 @@ export class AssessmentControllerService {
         schedule.disbursementDate,
       );
       disbursementDetails[`${disbursementIdentifier}Status`] =
+        schedule.disbursementScheduleStatus;
+      disbursementDetails[`${disbursementIdentifier}COEStatus`] =
         schedule.coeStatus;
       disbursementDetails[`${disbursementIdentifier}MSFAANumber`] =
         schedule.msfaaNumber.msfaaNumber;
+      disbursementDetails[`${disbursementIdentifier}MSFAACancelledDate`] =
+        schedule.msfaaNumber.cancelledDate;
+      disbursementDetails[`${disbursementIdentifier}MSFAADateSigned`] =
```

Comment:
Why do we send the date signed in API?, I couldn't see that in UI either in figma or screenshots.

![image](https://github.com/bcgov/SIMS/assets/54600590/fc10374b-77c0-41f0-a80c-cf9c00ea4ddd)

Reply:
The signed date received from the API is used by the method `getMSFAAStatus`.